extensions/ql-vscode/src/model-editor/languages/languages.ts

```diff
@@ -16,3 +19,16 @@ export function getModelsAsDataLanguage(
   }
   return definition;
 }
+
+export function getModelsAsDataLanguageModel<
+  T extends keyof ModelsAsDataLanguagePredicates,
+>(
+  language: QueryLanguage,
+  model: T,
+): NonNullable<ModelsAsDataLanguagePredicates[T]> {
+  const definition = getModelsAsDataLanguage(language).predicates[model];
+  if (!definition) {
+    throw new Error(`No models-as-data definition for ${model}`);
```

Comment:
```suggestion
    throw new Error(`No models-as-data predicate for ${model}`);
```
Should this error message be different from the error message from `getModelsAsDataLanguage` so it's easier to tell what's happened if we don't have the full stack trace.